src/site/stages/build/process-cms-exports/transformers/node-event_listing.js

```diff
@@ -20,7 +20,7 @@ const reverseFields = reverseFieldListing => ({
           entityPublished: reverseField.entityPublished,
           uid: reverseField.uid,
           fieldFeatured: reverseField.fieldFeatured,
-          fieldDate: reverseField.fieldDate,
+          fieldDatetimeRangeTimezone: reverseField.fieldDatetimeRangeTimezone,
```

Comment:
I noticed even though we're removing `fieldDate` from the transformer, it is still being pulled in the GraphQL [query](https://github.com/department-of-veterans-affairs/vets-website/blob/45c2c17d5b57f069cb6ecd94ba815f5bf4b474c2/src/site/stages/build/drupal/graphql/eventListingPage.graphql.js#L38). If we don't need `fieldDate` maybe we should remove it form the GraphQL query as well so we can have the outputs synced.

Also we probably need to add `fieldDatetimeRangeTimezone` to the output schema for `node-event_listing`, specifically in the [reverseFieldSchema](https://github.com/department-of-veterans-affairs/vets-website/blob/e32b2e277350bad6c8acf82868b7c79dad4636e2/src/site/stages/build/process-cms-exports/schemas/output/node-event_listing.js#L4).

Reply:
Fixed the schema issue. Not comfortable removing `fieldDate` from graphql until all issues with dates have been addressed.

Reply:
Agreed. Maybe we should leave `fieldDate` inside of the transformer to keep consistency since it's staying in the GraphQL for now?